dataclass_csv/dataclass_reader.py

```diff
@@ -13,13 +13,13 @@ class DataclassReader:
     def __init__(
         self,
         f,
```

Comment:
The `f` argument i missing annotation. I guess it could be something like `Iterable[str]`. What do you think?

Reply:
Whoops! I don't know why I didn't think of this before, but this should mirror DictReader in typeshed:

https://github.com/python/typeshed/blob/e3352c654d58ffba5951e14aa8972c4c6f17212b/stdlib/2and3/csv.pyi

Which suggests Iterable[Text] (which is basically the same thing as str in a python3 only world, so you are right, Iterable[str] is sufficient.